emcc.py

```diff
@@ -844,6 +844,9 @@ def optimizing(opts):
     lib_dirs = [shared.path_from_root('system', 'local', 'lib'),
                 shared.path_from_root('system', 'lib')]
 
+    # -c means do not link in gcc, and for us, the parallel is to not go all the way to JS, but stop at bitcode
```

Comment:
I would just say "-c mean compile to object files(s), don't link" since with the llvm backed we compile to wasm object files, not bitcode.